Podverse/PVDeleter.swift

```diff
@@ -18,25 +18,31 @@ extension Notification.Name {
 class PVDeleter: NSObject {
     
     static func deletePodcast(podcastId: NSManagedObjectID?, feedUrl: String? = nil) {
-        let moc = CoreDataHelper.createMOCForThread(threadType: .privateThread)
-        
-        if let podcastId = podcastId, let podcast = CoreDataHelper.fetchEntityWithID(objectId: podcastId, moc: moc) as? Podcast {
-            podcast.removeFromAutoDownloadList()
-            deleteAllEpisodesFromPodcast(podcast: podcast)
-            moc.delete(podcast)
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: .podcastDeleted, object: nil, userInfo: ["feedUrl": podcast.feedUrl as? Any])
+            let moc = CoreDataHelper.createMOCForThread(threadType: .mainThread)
```

Comment:
Why is this on the main thread if PVDeleter performs background tasks? Or does deletePodcast only run on the main thread...